samples/msal-node-samples/ElectronSystemBrowserTestApp/src/Graph.ts

```diff
@@ -0,0 +1,20 @@
+import { Client } from "@microsoft/microsoft-graph-client";
```

Comment:
In my opinion we should keep dev samples lean and straight to the point to better highlight the MSAL feature it is demonstrating. We can (and do) cover aspects like graph SDK usage etc. in quickstarts and tutorials.

Reply:
Removed and added fetch request with axios